x-pack/legacy/plugins/siem/public/components/ml_popover/types.ts

```diff
@@ -65,6 +173,7 @@ export interface SetupMlResponse {
 export interface StartDatafeedResponse {
   [key: string]: {
     started: boolean;
+    error?: string;
```

Comment:
Should `StopDatafeedResponse` below also optionally have an `error?:` and we just haven't added it like here yet?

Reply:
I've tried causing an error with `stop_datafeeds` API with our regular flow and was unsuccessful (would just receive an empty object back ). That said was able to receive the following when sending a bunk datafeedId:

```js
{
  statusCode: 500,
  error: 'Internal Server Error',
  message: 'An internal server error occurred',
};
```

I'll update the type to have an optional  `statusCode`, `error` and `message`.

Reply:
This won't work with `[key: string]` as a sibling type, so creating an `ErrorResponse` interface and using that in conjunction with `StopDatafeedResponse` and `CloseJobsResponse`.